apps/gitness/src/pages-v2/repo/transform-utils/__tests__/branch-transform.test.ts

```diff
@@ -0,0 +1,82 @@
+import { afterAll, beforeAll, describe, expect, it, vi } from 'vitest'
```

Comment:
You don't need these at all since they're already set up as globals
```suggestion
```

Reply:
done